src/components/sidebar/Sidebar.js

```diff
@@ -16,6 +17,21 @@ export default function Sidebar({
 }) {
   const drawerList = ['Home', 'Profile', 'Subscriptions'];
 
+  const collapses = {
```

Comment:
nit: can you make the name clear of what are these? One example I thought of is "collapsableSections".